src/stages/stagex/timeline.c

```diff
@@ -228,6 +228,73 @@ TASK(ngoner_fairy, { cmplx pos; }) {
 	e->move = move_linear(I);
 }
 
+static Boss *stagex_spawn_scuttle(cmplx pos0) {
+	Boss *scuttle = create_boss("Scutƫle", "scuttle", pos0);
+	boss_set_portrait(scuttle, "scuttle", NULL, "normal");
+	scuttle->shadowcolor = *RGBA(0.5, 0.0, 0.22, 1);
+	scuttle->glowcolor = *RGBA(0.30, 0.0, 0.12, 0);
+
+	return scuttle;
+}
+
+TASK(scuttle_appear, { cmplx pos; }) {
+	STAGE_BOOKMARK(midboss);
+	Boss *boss = global.boss = TASK_BIND(stagex_spawn_scuttle(ARGS.pos));
+
+	Attack *opening_attack = boss_add_attack(boss, AT_Normal, "Opening", 60, 40000, NULL, NULL);
+
+	boss_start_attack(boss, boss->attacks);
+	INVOKE_TASK(stagex_midboss_nonspell_1, ENT_BOX(boss), opening_attack);
```

Comment:
You probably want this:
```c
boss_add_attack_task(boss, AT_Normal, "Opening", 60, 40000, TASK_INDIRECT(BossAttack, stagex_midboss_nonspell_1), NULL);
boss_start_attack(boss, boss->attacks);
```

I'm guessing you copied this from Yumemi's first non. That one is a special case, because the attack task starts during the dialogue there, before the attack phase actually begins, as far as the boss system is concerned. It spawns some slaves, and waits for the attack's `initiated` event, to sync up with `boss_start_attack`. That's why the setup there is a little more complicated than usual.